roles/users/README.rst

```diff
@@ -0,0 +1,87 @@
+Users
+=====
+
+This role is used to manage user accounts on a node.  It is capable of setting up users
+with and without sudo access, but does not currently have the ability to remove users.
+In either your group_vars or host_vars files you can define two variables for this role
+to use: ``managed_users`` and ``managed_admin_users``.  The ``managed_users`` variable
+will create users without sudo access while users in the ``managed_admin_users`` list
+will be granted sudo access. Sudo access is granted by adding the ``managed_admin_users`` to
+the group ``sudo`` which should be created beforehand.
+
+For example, in inventory/group_vars/webservers.yml you might have a list of users like this::
```

Comment:
Seems better ot move this down to the vars section; then you don't have to repeat the bit about it being a string or a url, either